lib/src/theme/colors.dart

```diff
@@ -808,7 +806,7 @@ extension ZetaColorGetters on ColorScheme {
 
   /// Tertiary surface color.
   ///
-  /// Maps to [ColorScheme.background] and [ThemeData.scaffoldBackgroundColor]
+  /// Maps to [ColorScheme.surface] and [ThemeData.scaffoldBackgroundColor]
```

Comment:
After the deprecation of background color, this no longer corresponds directly to ColorScheme.surface; it now aligns only with scaffoldBackground color. According to Zeta foundations, surfacePrimary is what maps with ColorScheme.surface.